.chloggen/munir_add-option-to-avoid-remapping.yaml

```diff
@@ -0,0 +1,13 @@
+change_type: enhancement 
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Adds a featuregate that controls remapping metrics to match Datadog semantics"
```

Comment:
This release note is not informative enough. We should:
- List the name of the feature gate
- Explain more clearly what the feature gate does in terms understandable by an end-user
- Explain under what circumstances would one want to use the feature gate